configs/TCMM405V3/config.h

```diff
@@ -27,13 +27,15 @@
 
 #define USE_ACC
 #define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
```

Comment:
Do we know this IMU has a dedicated LDO :upside_down_face:

Reply:
what brand is it? I dont recognise the tag

Reply:
might be defunct. weird website.
![image](https://github.com/user-attachments/assets/f7162e39-ec24-4943-b953-9645c737c6cf)
http://www.tcmm-rc.cn/